### ISSUE TITLE
Configurable file extensions

### ISSUE THREAD

**Issue by dpfavand** [kaicataldo/eleventy-plugin-react#13]
Can we consider making file extensions configurable? Particularly thinking for cases where we want to handle TSX files, but not JS files.

I'd imagine adding an `extensions` property to the configuration object, an array of extensions that would override defaults:

```js
  eleventyConfig.addPlugin(eleventyReact, {
    extensions: ['tsx']
  });
```

**Comment by kaicataldo** [comment]
This should be easy to implement. PRs welcome!

**Comment by kaicataldo** [comment]
Implemented in  https://github.com/kaicataldo/eleventy-plugin-react/commit/d1fec2a1553ea454da852f323441841b70d63970 and released in v0.11.0.
